tket2/src/optimiser/taso/qtz_circuit.rs

```diff
@@ -145,7 +145,7 @@ mod tests {
     #[cfg_attr(miri, ignore)] // Opening files is not supported in (isolated) miri
     fn test_read_rep() {
         let rep_map: HashMap<String, Circuit> =
-            load_representative_set("test_files/h_rz_cxrepresentative_set.json");
+            load_representative_set("../test_files/h_rz_cxrepresentative_set.json");
```

Comment:
can we set some kind of workspace environment variable or something to make this less flaky?

Reply:
That will be a bit noisy, so I'll open an issue for that.